openslides/core/static/js/core/base.js

```diff
@@ -144,13 +145,22 @@ angular.module('OpenSlidesApp.core', [
             // set current language and return updated languages object array
             setCurrentLanguage: function (lang) {
                 var languages = this.getLanguages();
+                var plugins = OpenSlidesPlugins.getAll();
                 angular.forEach(languages, function (language) {
                     language.selected = false;
                     if (language.code == lang) {
                         language.selected = true;
                         gettextCatalog.setCurrentLanguage(lang);
                         if (lang != 'en') {
                             gettextCatalog.loadRemote("static/i18n/" + lang + ".json");
+                            // load language files from plugins
+                            if (plugins.length > 0) {
```

Comment:
I think this check is not necessary. If the length is 0 then the forEach will skip automaticly, right?